modules/crafted-evil-config.el

```diff
@@ -81,7 +81,13 @@ Rebinds the arrow keys to display a message instead."
                 term-mode))
   (add-to-list 'evil-emacs-state-modes mode))
 
-(evil-collection-init)
+(when (locate-library "evil-collection")
+  ;; Initialize evil-collection
+  (evil-collection-init)
+
+  ;; If speedbar is loaded, run evil-collection setup
+  (with-eval-after-load 'speedbar
+    (evil-collection-speedbar-setup)))
```

Comment:
If I understood the docs of `evil-collection` correctly, then `evil-collection-init` already encompasses `evil-collection-speedbar-setup`. So that code can be removed.